src/sql/parts/notebook/cellViews/code.component.ts

```diff
@@ -78,13 +79,15 @@ export class CodeComponent extends AngularDisposable implements OnInit, OnChange
 
 	protected _actionBar: Taskbar;
 	private readonly _minimumHeight = 30;
+	private readonly _maximumHeight = 4000;
```

Comment:
can we reconcile this max height and the one in queryTextEditor?

Reply:
Yeah 4000 can be our default.